### PR TITLE
Updates to web

### PR DESCRIPTION
**Pull request by JasminFarhi**
Added Mesh photos
Added careers page and apply
Added updates to agile principals

### CODE REVIEW

**Review by xiangzhe0812 — APPROVED** [review]

**Comment by xiangzhe0812** [comment]
just neet format few pages like careers.html

File: theMesh.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                                <div class="core col-sm-6">
+                                  <div class="fh5co-post to-animate">
+                                    <meta name="viewport" content="width=device-width, initial-scale=1">                                                                                                  
+                                    <div>
+                                      <div class="grid-container">
+                                        <div class="div-text">
+                                          <div class="circular--landscape">                                    
+                                            <img src="../dltxio.github.io/images/The-Mesh/Alex-Xiang.jpg" alt="">                                                                                                                          
+                                          </div>
+                                         Alex Xiang <br>
```
Comment (Mitchell139) [inline theMesh.html:89]: Formatting

File: theMesh.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                                        <div class="div-text">       
+                                        <div class="circular--landscape">
+                                          <img src="../dltxio.github.io/images/The-Mesh/Lorenzo-Fontoura.jpg" alt="">
+                                        </div>
+                                        Lorenzo Fontoura <br>
+                                        Developer 
+                                        </div> 
+                                        <div class="div-text">
+                                        <div class="circular--landscape">
+                                          <img src="../dltxio.github.io/images/The-Mesh/Mitchel-Westmoreland.jpg" alt="">
```
Comment (Mitchell139) [inline theMesh.html:115]: That's not how you spell my name

File: theMesh.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                                        <div class="div-text">               
+                                        <div class="circular--landscape">
+                                          <img src="../dltxio.github.io/images/The-Mesh/Scott-Drummond.jpg" alt=""> 
+                                        </div>
+                                        Scott Drummond <br>
+                                        Developer 
+                                        </div>        
+                                        <div class="div-text">               
+                                        <div class="circular--landscape">
+                                          <img src="../dltxio.github.io/images/The-Mesh/Rob Braunack.jpg" alt="">
```
Comment (Mitchell139) [inline theMesh.html:143]: Missing hyphen

**Review by Mitchell139 — CHANGES_REQUESTED** [review]
What's the point of the "theMesh.md"? Right now it's empty

File: theMesh.html (hunk 2 above)
Comment (bitcoinbrisbane) [reply theMesh.html]: lol

Comment (bitcoinbrisbane) [reply theMesh.html]: Fixed

File: theMesh.html (hunk 3 above)
Comment (bitcoinbrisbane) [reply theMesh.html]: Fixed.

File: theMesh.html (hunk 1 above)
Comment (bitcoinbrisbane) [reply theMesh.html]: Fixed

File: careers.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+              <h2 class="to-animate">Interested in joining the mesh? Register your interest below <br> We are looking
+                for...</h2>
+            </div>
+            <section id="fh5co-clients" data-section="clients">
+              <div class="core col-sm-6">
+                <div class="fh5co-post to-animate">
+                  <meta name="viewport" content="width=device-width, initial-scale=1">
+                  <div class="grid-careers">
+                    <div class="circular--careers">
+                      <img src="../dltxio.github.io/images/icons/web-development.png" alt="">
```
[the remark this answers is not in the document]
Comment (bitcoinbrisbane) [reply careers.html]: Fixed

File: careers.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+      <!-- Waypoints -->
+      <script src="js/jquery.waypoints.min.js"></script>
+      <!-- Stellar Parallax -->
+      <script src="js/jquery.stellar.min.js"></script>
+      <!-- Counters -->
+      <script src="js/jquery.countTo.js"></script>
+      <!-- Main JS (Do not remove) -->
+      <script src="js/main.js"></script>
+</body>
```
[the remark this answers is not in the document]
Comment (bitcoinbrisbane) [reply careers.html:149]: Fixed

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**